File: JavaScript/8js 2/man_woman.js
/*
|--------------------------------------------------------------------------
| Basic subclasses - Adam and Eve
|--------------------------------------------------------------------------
|
*/

class God {
   static create() {
      return [new Man(), new Woman()];
   }
}
class Human {}
class Man extends Human {}
class Woman extends Human {}


class God {
   static create() {
      return [new Man("Adam"), new Woman("Eve")];
   }
}
class Human {
   constructor(name) {
      this.name = name;
   }
}
class Man extends Human {
   constructor(name) {
      super(name);
   }
}
class Woman extends Human {
   constructor(name) {
      super(name);
   }
}

File: Python/8py 2/man_woman.py
"""
|--------------------------------------------------------------------------
| Basic subclasses - Adam and Eve
|--------------------------------------------------------------------------
|
"""

def God():
   return [Man(), Woman()]
class Human: pass
class Man(Human): pass 
class Woman(Human): pass


God, Human = lambda: [Man(), Woman()], type('', (), {})
Man = Woman = type('', (Human,), {})


Human = type('Human', (), {})
Man = type('Man', (Human,), {})
Woman = type('Woman', (Human,), {})
def God():
   return [Man(), Woman()]